recipes/libdicom/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libdicom" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ImagingDataCommons/libdicom/releases/download/v{{ version }}/libdicom-{{ version }}.tar.xz
+  sha256: 3b88f267b58009005bc1182d8bd0c4a3218013ce202da722e5e8c9867c6f94f4
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson
+    - pkg-config
+  host: []
+  run: []
+
+test:
+  commands:
+    - test -f $PREFIX/lib/*/libdicom${SHLIB_EXT}  # [unix]
```

Comment:
can we be more precise about exactly where this ends up. i don't like the `*`.

Reply:
i also don't like the *. on linux, the directory is lib/x86_64-linux-gnu/. not sure what it is on macOS though.